go.mod

```diff
@@ -1,18 +1,28 @@
 module github.com/bitrise-io/go-android/v2
 
-go 1.16
+go 1.22.10
```

Comment:
This won't work with Go 1.22.9 and below according to the go module rules. Can we only define `go 1.22`?